src/interface/misc.rs

```diff
@@ -204,6 +204,9 @@ pub fn sigcheck() -> bool {
 }
 
 pub fn fillrandom(bufptr: *mut u8, count: usize) -> i32 {
```

Comment:
I think this comment should clarify that for now we don't care about the blocking/entropy aspect of /dev/random